docs/community/contribute/compliance.md

```diff
@@ -0,0 +1,151 @@
+
+```
+spec:
+  id: eks-cis
+  title: AWS EKS CIS Foundations v1.4
+  description: AWS EKS CIS Foundations
+  version: "1.4"
+  relatedResources:
+  - https://www.cisecurity.org/benchmark/amazon_web_services
+  controls:
+  - id: 2.1.1
+    name: Enable audit Logs (Automated)
+    description: |
+      Control plane logs provide visibility into operation of the EKS Control plane components ystems. 
+      The API server audit logs record all accepted and rejected requests in the cluster. 
+      When enabled via EKS configuration the control plane logs for a cluster are exported to a CloudWatch 
+      Log Group for persistence.
+    checks: null
+    severity: MEDIUM
+  - id: 3.1.1
+    name: Ensure that the kubeconfig file permissions are set to 644 or more restrictive (Manual)
+    description: |
+      If kubelet is running, and if it is configured by a kubeconfig file, ensure that the proxy kubeconfig 
+      file has permissions of 644 or more restrictive
+      Check with the following command: 
+      > sudo systemctl status kubelet
+    checks: null
+    severity: HIGH
+  - id: 3.1.2
+    name: Ensure that the kubelet service file ownership is set to root:root (Manual)
+    description: Ensure that the kubelet service file ownership is set to root:root
+    checks:
+      - id: AVD-KCV-0070
+    severity: HIGH
+    ....
+```
+
+The first section in the spec is focused on the metadata of the spec. Replace all the fields of the metadata with the information relevant to the compliance spec that will be added. This information can be taken from the official report.
+
+- `id` -- Required; this ID is composed of the resource that this check focuses on and the type of compliance spec
+- `title` -- Required; a short title of the spec
+- `description` -- Required; this can be a longer description of the compliance spec
+- `version` -- Required; the compliance spec version that the compliance spec targets. For instance, for CIS Benchmarks version 1.4, the version will be `1.4`.
+- `relatedResources` -- Any resources that relate to this compliance spec and can provide users with more information in the future.
+
+The second section details all of the checks that make up the compliance spec. These checks are detailed under `controls`.
+
+  - `id` -- Required; this has to be the ID from the official compliance documentation such as the CIS Benchmark version.
+  - `name` -- Required; this is the name of the check in short format.
+  - `description` -- Required; this is a description of how the resource should be configured.
+  - `checks.id` -- Required; this is the AVD ID or AVD IDs referenced that perform the Rego check for this compliance check, more information is provided below.
+  - `severity` -- Required; more information provided below.
+
+### Populating the `control` section
```

Comment:
there's more to say about vulnerability checks and node checks

Reply:
@chen-keinan :) help